site/en/blog/whats-new-css-ui-2023/index.md

```diff
@@ -0,0 +1,669 @@
+
+### Nesting
+
+{% BrowserCompat 'css.at-rules.nest' %}
+
+CSS nesting, something folks love from Sass, and one of the top CSS developer requests for years, is finally landing on the web platform. Nesting allows developers to write in a more succinct, grouped format that reduces redundancy.
+
+```css
+.card {}
+.card:hover {}
+
+/* can be done with nesting like */
+.card {
+  &:hover {
+    
+  }
+}
+```
+
+You can also nest [Media Queries](/learn/design/media-queries/), which also means you can nest [Container Queries](https://web.dev/cq-stable/). In the following example, a card is changed from a portrait layout to a landscape layout if there's enough width in it's container:
+
+```css
+.card {
+  display: grid;
+  gap: 1rem;
+
+  @container (width >= 480px) {
+    display: flex;
+  }
+}
+```
+
+The layout adjustment to `flex` occurs when the container has more (or equal to) `480px` of inline space available. The browser will simply apply that new display style when the conditions are met.
+
+For more information and examples, check out our post on [CSS nesting](/articles/css-nesting/).
+
+### Cascade layers 
+
+{% BrowserCompat 'css.at-rules.layer' %}
+
+Another developer pain point we’ve identified is ensuring consistency in which styles win over others, and one part of resolving this is having better control over the CSS cascade.
+
+[Cascade layers](/blog/cascade-layers/) solve this by giving users control over which layers have a higher precedence than others, meaning more fine-tuned control of when your styles are applied. 
+
+{% Img src="image/HodOHWjMnbNw56hvNASHWSgZyAf2/jH8ydtcwmHg4rHTfqvRU.png", alt="Cascade Illustration", width="800", height="393" %}
+
+<figure>
+    {% Img src="image/HodOHWjMnbNw56hvNASHWSgZyAf2/CSUb8jZLi4QJUdmTM109.png", alt="Screenshot from Codepen Project", width="800", height="556" %}
+    <figcaption>Explore the <a href="https://codepen.io/web-dot-dev/project/editor/ZGQLkq">project on Codepen.</a>
+    </figcaption>
+</figure>
+
+Learn more about how to use cascade layers [in this article](/blog/cascade-layers/).
+
+### Scoped CSS 
+
+{% BrowserCompat 'css.at-rules.scope' %}
+
+
+CSS scoped styles allow developers to specify the boundaries for which specific styles apply, essentially creating native namespacing in CSS. Before, developers relied on 3rd party scripting to rename classes, or specific naming conventions to prevent style collision, but now, you can use @scope. 
+
+Here, we’re scoping a .title element to a card. This would prevent that title element from conflicting with any other elements on the page that have a class of title, like a blog post title or other heading.
+
+Example -
```

Comment:
GitHub is having issues with nesting a code block in a suggestion, but an example can be found at https://docs.google.com/presentation/d/1wl1gurd1jk9xUPmX9kQo44i4dB1488ER5TVa9lV9efQ/edit?pli=1#slide=id.g232b9071b03_6_894